index.html

```diff
@@ -51,6 +51,109 @@ <h1 class="introduction">I'm Ivan<br>Glad to see you</h1>
             </div>
         </div>
     </section>
+    <section class="works">
+        <div class="work">
+            <img src="./images/work1.png" alt="screenshot of first project" class="work1_pic">
+            <h3 class="work-name">Tonic</h3>
+            <div class="work-data">
+                <strong class="company"> Canopy </strong>
+                <img src="./images/Counter.png" class= "counter" alt="counter">
+                <strong class="role">Back-End Dev</strong>
+                <img src="./images/Counter.png" class= "counter" alt="counter">
+                <strong class="year">2015</strong>
+            </div>
+            <p class= "paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class ="tags">
+                <img src="./images/html.png" alt="html tag" class= "work-tags">
+                <img src="./images/css.png" alt="css tag" class= "work-tags">
+                <img src="./images/javascript.png" alt="javascript tag" class= "work-tags">
```

Comment:
- [ ] Kindly change this `<img>` tags and put them inside a `<ul>` tag with the correspondent `<li>` tag. This will help you keep the code cleaner and more semantic.
- [ ] Also, in order to follow [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) please remove the blank spaces in `class= "work-tags"` so it becomes `class="work-tags"`